README.md

```diff
@@ -16,6 +16,12 @@ Cumulus protocol to a format more easily consumable by Cumulus tasks, invokes
 the tasks, and then adapts their response back to the Cumulus message protocol
 to be sent to the next task.
 
+## Installation
+
+The cumulus-message-adapter-js can be installed via Node Package Manager (NPM) and the package is located [here](https://www.npmjs.com/package/@cumulus/cumulus-message-adapter-js).
+
+Add @cumulus/cumulus-message-adapter-js to your package.json and run ```npm install```.
```

Comment:
The package can be added to your project by running `npm install @cumulus/cumulus-message-adapter-js --save`.

Reply:
Done.